libs/filesystem/src/main/java/org/padaiyal/utilities/filesystem/parameterconverters/ExceptionClassConverter.java

```diff
@@ -1,4 +1,4 @@
-package org.padaiyal.mavenprojecttemplate.parameterconverters;
+package org.padaiyal.utilities.filesystem.parameterconverters;
```

Comment:
Refer to the unittestextras package which has this class. This file can be removed.